docs/ui-overview.md

```diff
@@ -1,18 +1,20 @@
 # UI Overview
 
-![UI Overview](../assets/images/ui-overview.png)
+This article provides a high-level overview of the Stoplight Studio User Interface (or UI). Below is a screenshot of the UI with the different sections labeled below.
 
-1. **Project Selector**: Displays the currently selected project. Click to select a different project from the dropdown menu, search for a project, or create a new project. 
+![Overview of the Studio User Interface](../assets/images/ui-overview.png)
 
-2. **Project Actions**: Contains actions to help you add new assets to your project. This includes creating new: APIs, Endpoints, Models, Articles, Images, Files, Folders, and Projects. 
+1. **Project Name/Selector**: Displays the name of the current project. Click the drop-down to select a different project, search for an existing project, or create a new one.
 
-3. **File tree**: Displays all Project files. Can toggle between **APIs** file tree (APIs, endpoints, and models), **Docs** file tree (markdown and image files), **Files** file tree (all files), and **Git** (not displayed, Pull/Push Git functionality). 
+2. **Project Actions**: Contains actions to help you add new assets to your project. This includes creating new: APIs, Endpoints, Models, Articles, Images, Files, Folders, and Projects.
 
-4. **Primary Panel**: Main editor panel. Can switch between view types via the view toggle in the top right of the panel (not shown, must hover over panel for toggle to appear).
+3. **Project Sidebar**: Displays an overview of the project in different ways. Can toggle between the **APIs** view (displays APIs, endpoints, and models), the **Docs** view (for markdown and image files), the **Files** view (displaying all project files), and a **Git** view (for pushing/pulling changes to the connected Git repository).
```

Comment:
not sure if we want to use "view" to describe the different file trees, could become confusing

Reply:
Removed view